### PR TITLE
Split "change formats" into separate categories

### PR DESCRIPTION
**Pull request by pjotrek-b**
Related to suggestion #169.

The "mkv to mp4" rewrap command feels a bit alone now, but probably not for long :wink:

### CODE REVIEW

**Review by retokromer — APPROVED** [review]

**Comment by retokromer** [comment]
It’s fine that way!

File: index.html [hunk 1]
```diff
@@ -476,6 +311,188 @@
   </div>
   <!-- ends Transcode to H.265 -->
 
+  <p>&nbsp;</p>
```
Comment (retokromer) [inline index.html:314]: Why is this necessary?

**Review by retokromer — COMMENTED** [review]
@pjotrek-b I missed this:

File: index.html (hunk 1 above)
Comment (pjotrek-b) [reply index.html:314]: To make an optical gap between video and audio-only commands.
First wanted to introduce an "audio" headline, but that looked bloated...
